pkg/build/api/v1beta3/types.go

```diff
@@ -385,6 +390,9 @@ type BuildRequest struct {
 
 	// Revision is the information from the source for a specific repo snapshot.
 	Revision *SourceRevision `json:"revision,omitempty"`
+
+	// Image is the image that triggered this build.
+	Image string `json:"image,omitempty"`
```

Comment:
@liggitt i believe that this field is set to image pull spec (DockerImageReference), but @ncdc can prove me wrong.